lib/cc/engine/fingerprint.rb

```diff
@@ -33,7 +33,12 @@ def compute
       attr_reader :path, :cop_name, :message
 
       def stripped_message
-        message.gsub(/ \[.+\]$/, "")
+        # example full message with details:
+        # Metrics/MethodLength: Method has too many lines. [18/10]  (https://github.com/bbatsov/ruby-style-guide#short-methods)
```

Comment:
I think the spec is enough and don't think this comment is hugely valuable. WDYT?